packages/neo-one-website/src/components/docs/AdjacentLink.tsx

```diff
@@ -0,0 +1,53 @@
+import * as React from 'react';
+import { Link as RouterLink } from 'react-static';
+import { Box, styled } from 'reakit';
+import { prop } from 'styled-tools';
+import { AdjacentInfo } from '../../utils';
+
+export interface Props {
+  readonly adjacent: AdjacentInfo;
+  readonly next: boolean;
+}
+
+const ArticleText = styled(Box)`
+  ${prop('theme.fonts.axiformaRegular')};
+  color: ${prop('theme.gray0')};
+
+  @media (max-width: ${prop('theme.breakpoints.md')}) {
+    font-size: 12px;
+  }
+`;
+
+const NavigationLink = styled(RouterLink)`
+  display: flex;
+  align-items: center;
+  ${prop('theme.fonts.axiformaRegular')};
+  font-size: 24px;
+  height: 100%;
+  width: 100%;
+  color: ${prop('theme.primary')};
+
+  &:hover {
+    border-color: ${prop('theme.accent')};
+    color: ${prop('theme.accent')};
+  }
+
+  &.active {
+    border-color: ${prop('theme.accent')};
+  }
+
+  @media (max-width: ${prop('theme.breakpoints.md')}) {
+    font-size: 16px;
+  }
+`;
```

Comment:
Actually, this should really just use the `Link` component from `@neo-one/react-common` with `linkColor="primary"`

Reply:
still needs fixing